.github/workflows/go.yml

```diff
@@ -0,0 +1,69 @@
+name: Go
```

Comment:
it is better to call it by function, what workflow does

Reply:
renamed to build